aws-ssm-patchbaseline/src/test/java/software/amazon/ssm/patchbaseline/ListHandlerTest.java

```diff
@@ -100,12 +102,16 @@ public void handleRequest_SimpleSuccessWithNoProperty() {
         final ProgressEvent<ResourceModel, CallbackContext> response =
                 listHandler.handleRequest(proxy, request, null, logger);
 
+        ResourceModel expectedModel = ResourceModel.builder().id("pb-12345678901234567").operatingSystem(null).description(null).name(null).build();
+
         assertThat(response).isNotNull();
         assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
         assertThat(response.getResourceModel()).isNull();
         assertThat(response.getMessage()).isNull();
         assertThat(response.getErrorCode()).isNull();
-
+        if(response.getResourceModels().size() > 0){
```

Comment:
why does this have to be greater than 0? have you confirmed that in the test case, this is actually > 0?

Reply:
in your code implementation, is operatingSystem/description actually return null value? you said earlier it's empty string

Reply:
and i still don't know why this new test case compare with the first test case, i don't see material change between those 2 test cases

Reply:
if(response.getResourceModels().size() > 0){
            assertThat(response.getResourceModels().get(0)).isEqualTo(expectedModel);
        }
        
In the above snippet, After list handler invoking process, it will return the baseline details as list of values, so I have verifying if the list has elements or not. And I am getting first element from the list and validating the response with my expected model.

In the the describe model , we are creating model with only one baseline id process, so i am trying to get  that value in the list. So i am checking the list has elements or not